soh/src/overlays/actors/ovl_En_Wood02/z_en_wood02.c

```diff
@@ -351,7 +352,14 @@ void EnWood02_Update(Actor* thisx, PlayState* play2) {
             dropsSpawnPt = this->actor.world.pos;
             dropsSpawnPt.y += 200.0f;
 
-            if ((this->unk_14C >= 0) && (this->unk_14C < 0x64)) {
+            if ((this->unk_14C >= 0) && (this->unk_14C < 0x64) && (CVarGetInteger("gTreeStickDrops", 0)) && !(INV_CONTENT(ITEM_STICK) == ITEM_NONE)) {
+                srand(time(NULL));
+                (numDrops = (rand() % 4));
```

Comment:
@jbodner09 what would the best option here be? I recall we wanted to move towards using random.hpp for everything but I don't think we can import that in C

Reply:
All vanilla gameplay logic uses the pseudo-rand functions in `code_800FD970`, so you should use the same since this is being added to a vanilla file.  Specifically, you'll want `Rand_ZeroOne() * 4` to get your numDrops.  

For the upgrades, my plan is to have LUS provide everything and use the OTRGlobals as a bridge to C.  They'll then upgrade those Rand functions behind the scenes so the callers won't need to make any changes.

Reply:
Worked like a charm! Thanks @jbodner09